plugins/obs-filters/CMakeLists.txt

```diff
@@ -152,6 +166,13 @@ if(NOT OS_MACOS)
     FILES ${_FILTERS})
 endif()
 
+if(LIBNVVFX_FOUND)
+  target_sources(obs-filters PRIVATE nvidia-greenscreen-filter.c)
+  obs_status(
+    STATUS"\nNVidia Video FX support enabled; requires redist to be installed by end-user"
```

Comment:
You still need a space between "STATUS" and the string.

Reply:
done

Reply:
cmake-format removes the space ...